Assets/MixedRealityToolkit.Tests/MixedReality.Toolkit.Tests.nuspec

```diff
@@ -14,14 +14,15 @@
     <releaseNotes>$releaseNotes$</releaseNotes>
     <tags>Unity MixedReality MixedRealityToolkit MRTK</tags>
     <dependencies>
+      <dependency id="Microsoft.MixedReality.Toolkit.Tests.Utilities" version="$version$" />
       <dependency id="Microsoft.MixedReality.Toolkit.Foundation" version="$version$" />
     </dependencies>
     <contentFiles>
       <files include="any\any\.PkgSrc\**" buildAction="None" copyToOutput="false" />
     </contentFiles>
   </metadata>
   <files>
-    <file src="**" exclude="*.nuspec;*.nuspec.meta;*.props;*.props.meta;*.targets;*.targets.meta" target="MRTK\" />
+    <file src="**" exclude="*.nuspec;*.nuspec.meta;*.props;*.props.meta;*.targets;*.targets.meta; TestUtilities\**" target="MRTK\" />
```

Comment:
I'm trying to leave a comment on line 29/30, but I can't seem to figure out how to do so (no changes there).

I think that this:
```
 <file src="..\..\Plugins\**\Microsoft.MixedReality.Toolkit.Tests.*" target="Plugins\" />
```

Needs to be updated to exclude the Test.Utilities binary right? Otherwise it'll end up getting included in both packages.

Reply:
Same comment for the test utilities CS code.

Reply:
Oh yes, you are right.  The test utilities were included in the test package when the test package should only contain the play/edit mode tests.
Added this to exclude the TestUtilities from the Tests package:
```
<file src="..\..\Plugins\**\Microsoft.MixedReality.Toolkit.Tests.*" exclude="..\..\Plugins\**\Microsoft.MixedReality.Toolkit.Tests.Utilities*" target="Plugins\" />
```